src/Tools/NodeDocumentationMarkdownGenerator/Verbs/FromDirectoryOptions.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System.Collections.Generic;
+using CommandLine;
+using CommandLine.Text;
+
+namespace NodeDocumentationMarkdownGenerator.Verbs
+{
+    [Verb("fromdirectory", HelpText = "Generate documentation from a directory containing binary files and dyfs")]
+    internal class FromDirectoryOptions
+    {
+
+        [Option('i', "input", HelpText = "Directory folder path", Required = true)]
+        public string InputFolderPath { get; set; }
+
+        [Option('o', "output", HelpText = "Folder path to save generated documents in", Required = true)]
+        public string OutputFolderPath { get; set; }
+
+        [Option('r', "references", HelpText = "Folder paths to dlls that are used as references in the nodes", Required = false)]
+        public IEnumerable<string> ReferencePaths { get; set; }
```

Comment:
this says folder paths, is it paths to folders of binaries or to binaries directly?

Reply:
paths to folders containing the binaries. Updated the help text to reflect that